ProblemSolvingMentoringSession/Part 1/FindExtremumsInListChallenge/Solvings/FindExtremumsInListChallenge_Yakunin.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  FindExtremumsInListChallenge_Yakunin.swift
+//  ProblemSolvingMentoringSession
+//
+//  Created by Yefim on 02.05.2023.
+//  Copyright © 2023 Almost Engineer. All rights reserved.
+//
+
+import Foundation
+final class FindExtremumsInListChallenge_Yakunin: FindExtremumsInListChallengeProtocol {
+    func solve(list: [Int]) -> FindExtremumsInListChallengeResult {
+        guard !list.isEmpty else { return FindExtremumsInListChallengeResult(minimum: nil, maximum: nil) }
+        return FindExtremumsInListChallengeResult(minimum: list.min(), maximum: list.max())
```

Comment:
rewrite it please on custom for in logic for searching extremum, without min max built in functions

Reply:
Done